justfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018-2020 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version := "0.4"
+
+copy-docs:
+    #!/usr/bin/env sh
+    if test -f "../splinter/Cargo.toml"; then
+        # Print the branch of splinter the man pages are from and the final
+        # location
+        splinter_branch=$(cd ../splinter && git name-rev --name-only HEAD)
+        echo "Syncing man pages from $splinter_branch to docs/{{version}}"
+
+        # sync man pages from splinter cli and splinterd
+        rsync -r ../splinter/cli/man/ docs/{{version}}/references/cli
```

Comment:
Adding -v here would output the changed files, etc.

Reply:
added